### ISSUE TITLE
CSS is broken/ignored since version 10 (or 9)

### ISSUE THREAD

**Issue by Araly** [MichaelAquilina/improved-workspace-indicator#13]
Since an update a couple days ago, my `.local/share/gnome-shell/extensions/improved-workspace-indicator@michaelaquilina.github.io/stylesheet.css` is not taken into account anymore, and the workspace indicator looks like the images on this repo instead of what I used to have.

I'm using Gnome Wayland 41.3, on Arch

Not sure if there are logs I can bring up, if I should move my file somewhere else or anything.

**Comment by MichaelAquilina** [comment]
I would be really surprised if this is due to any changes in the release. I suspect it's more likely to be due to gnome updating the extension for you and no longer using the directory you are referring to. If the CSS file was changed, then the extension would have to also reflect what's there.

Unfortunately I don't know enough about the gnome shell extension ecosystem to help more than that. Please post if you find anything relevant that might help other users

**Comment by Araly** [comment]
yeah I was thinking maybe the extension was renamed or the versioning changes where the file should go, but checking in my file system I don't see where things would go instead. The file is still there and I don't see anything that would have changed. Maybe gnome changed in a way that CSS doesn't work for extensions anymore ? this is the only extension I have CSS for
